examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/BasicClientFragment.kt

```diff
@@ -150,13 +153,13 @@ class BasicClientFragment : Fragment() {
           }
 
           override fun onReport(nodeState: NodeState?) {
-            val value =
+            val tlv =
               nodeState
                 ?.getEndpointState(endpointId)
                 ?.getClusterState(clusterId)
                 ?.getAttributeState(attributeId)
-                ?.value
-                ?: "null"
+                ?.tlv
+            val value = tlv?.let { TlvReader(it).toAny() }
```

Comment:
It is not clear to me what toAny() do here, since we already have the TLV object, could we just convert it to json using the tlvjson convertor API we have?

Reply:
@yufengwangca 
The purpose of this API is to ensure that simple values (int, double, float, string etc.) are not expressed in json format. (Of course, if tlv is a struct type, it is displayed in json format.) Conversely, tlvwriter.put() can be used to change the simple value to tlv.

Reply:
It look we plan to put all attributes/events under a struct after this change, 
 https://github.com/project-chip/connectedhomeip/pull/28639/files#diff-dff49ad6b12b07080eceb570603014fffffb05a02ee3bf2618a72eda72dfff3bR283
 
 Do we still have case the value is expressed in a simple format  (int, double, float, string etc.) ?

Reply:
Got it, the problem is I don't know what toAny really do here, it looks it try to encode the TLV into a object, right? Or it is used to transform the TLV into an more readable format?

Reply:
This method is only used in ChipTool. For this reason, I moved to this code to ChipTool App.

Reply:
thanks